src/main/scala/com/cognite/sdk/scala/v1/resources/dataPoints.scala

```diff
@@ -494,28 +520,18 @@ object DataPointsResource {
   implicit val cogniteExternalIdItemsEncoder: Encoder[Items[CogniteExternalId]] = deriveEncoder
   implicit val dataPointDecoder: Decoder[DataPoint] = deriveDecoder
   implicit val dataPointEncoder: Encoder[DataPoint] = deriveEncoder
-  implicit val dataPointsByIdResponseDecoder: Decoder[DataPointsByIdResponse] = deriveDecoder
-  implicit val dataPointsByIdResponseItemsDecoder: Decoder[Items[DataPointsByIdResponse]] =
-    deriveDecoder
-  implicit val dataPointsByExternalIdResponseDecoder: Decoder[DataPointsByExternalIdResponse] =
-    deriveDecoder
-  implicit val dataPointsByExternalIdResponseItemsDecoder
-      : Decoder[Items[DataPointsByExternalIdResponse]] =
+  implicit val dataPointsByIdResponseDecoder: Decoder[DataPointsResponse] = deriveDecoder
```

Comment:
Similarly for those decoders, I think they were used only to decode JSON responses to data points, and now we use protobuf instead.